controllers/apply/get-advice/fields.js

```diff
@@ -19,16 +19,16 @@ const {
     NameField
 } = require('../lib/field-types');
 
-const locationsFor = require('./lib/locations');
+const locationOptions = require('../lib/location-options');
 
 module.exports = function fieldsFor({ locale, data = {} }) {
     const localise = get(locale);
 
-    const projectCountries = getOr([], 'projectCountry')(data);
+    const projectCountries = getOr([], 'projectCountries')(data);
 
-    function fieldProjectCountry() {
+    function fieldProjectCountries() {
         return new CheckboxField({
-            name: 'projectCountry',
+            name: 'projectCountries',
```

Comment:
Renamed this to `projectCountries` to reflect that multiple options can be selected and this is an array